src/plugins/embeddable/public/lib/state_transfer/types.ts

```diff
@@ -40,6 +40,7 @@ export function isEmbeddableEditorState(state: unknown): state is EmbeddableEdit
 export interface EmbeddablePackageByReferenceState {
   type: string;
   id: string;
+  embeddableIdToReplace?: string;
```

Comment:
This doesn't have to go in this PR, but doesn't hurt. IMHO my opinion it would be better called just `embeddableId`, but as you wish.

Reply:
I will remove it from this PR, since it's no longer used. That key is actually just a workaround, specifically for removing the 'by value' embeddable after it has been added to the library. Once https://github.com/elastic/kibana/issues/70381 is fixed it should definitely be renamed `embeddableId`.